docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -780,6 +781,36 @@ directly or used in conditional statements.
 - `$file_size`
   : Size of the file for this row.
 
+### Replace tables
+
+The connector supports replacing a table, in case the table already exists, as
+an atomic operation. Atomic table replacement creates a new snapshot with the
+new table definition and keeps the existing table history. See
+[](#delta-lake-history-table) in metadata tables.
+
+To replace a table, use `CREATE OR REPLACE TABLE` and/or `CREATE OR REPLACE
+TABLE AS`. For more information on creating tables, see [](/sql/create-table)
+and [](/sql/create-table-as). These statements do not support changing table
+formats defined by the `type` table property.
+
+After replacement, the table definition is completely new and separate from the
+old table.
+
+For example, a table `example_table` can be replaced by a completely new
+definition:
+
+```
+CREATE TABLE example_table (
+    a BIGINT,
+    b DATE,
+    c BIGINT)
+WITH (partitioning = "partitioned_by");
+
+CREATE OR REPLACE TABLE example_table
```

Comment:
move this to separate code block and have intro sentence that this create new table with data